assets/templates/partials/census/get-data.tmpl

```diff
@@ -20,29 +20,33 @@
                     <span class="ons-radios__item ons-radios__item--no-border ons-u-mb-s">
                         <span class="ons-radio ons-radio--no-border">
                             <input type="radio" id="{{ .Extension }}" class="ons-radio__input ons-js-radio" value="{{ .Extension }}" name="format">
-                                <label class="ons-radio__label{{ if or (eq .Extension "txt") (eq .Extension "csvw") (eq .Extension "xls") }} ons-label--with-description{{ end }}" for="{{ .Extension }}" id="{{ .Extension }}-label"><span class="ons-u-tt-u">{{ .Extension }}</span> format ({{ humanSize .Size }})
-                                    <span id="{{ .Extension }}-label-description-hint" class="ons-label__description ons-radio__label--with-description">
-                                        {{ if eq .Extension "xls" }}
-                                            {{ localise "IncludesSupportingInfo" $.Language 1 }}
-                                        {{ else if or (eq .Extension "txt") (eq .Extension "csvw") }}
-                                            {{ localise "SupportingInfo" $.Language 1 }}
-                                        {{ end }}
-                                    </span>
-                                </label>
-                            </span>
+                            <label class="ons-radio__label{{ if or (eq .Extension "txt") (eq .Extension "csvw") (eq .Extension "xls") }} ons-label--with-description{{ end }}" for="{{ .Extension }}" id="{{ .Extension }}-label"><span class="ons-u-tt-u">{{ .Extension }}</span> format ({{ humanSize .Size }})
+                                <span id="{{ .Extension }}-label-description-hint" class="ons-label__description ons-radio__label--with-description">
+                                    {{ if or (eq .Extension "xls") (eq .Extension "xlsx") }}
+                                        {{- localise "IncludesSupportingInfo" $.Language 1 -}}
+                                    {{ else if eq .Extension "csv" }}
+                                        {{- localise "MachineReadable" $.Language 1 -}}&nbsp;<span class="ons-u-tt-l">{{- localise "Dataset" $.Language 1 -}}</span>
+                                    {{ else if eq .Extension "csvw" }}
```

Comment:
Is the &nbsp intentional? If so, wouldn't padding/margin be better here?

Reply:
I put it in to make the code a bit more readable and (hopefully) prevent the space from being removed accidentally. Happy to replace it with normal whitespace if you'd prefer

Reply:
I think normal whitespace would be better!